### DISCUSSION TITLE
Self Leveling Issue

### DISCUSSION THREAD

**Discussion by coreprat** [Jyers/Marlin#1514] [Q&A] [answered]
Has anyone come across this issue? 

When the auto leveling occurs, the extruder will not move left on spots 5 and 6?

This will cause it to hit the frame on spots 8 and 9.

**Comment by coreprat** [comment]
For Video https://www.reddit.com/r/ender3v2/comments/qeleeu/has_anyone_come_across_this_issue_when_the_auto/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by Jyers** [comment]
Try resetting eeprom. Go to control, select reset to defaults and then store settings. If that doesn't work, I'll have a new release out soon that you could try.

**Comment by coreprat** [comment, marked as answer]
Kinda feel like an idiot… come to find out I totally eliminated the X cable between the x stepper motor and the board.

/me pie on face

**Reply by Jyers** [reply]
That'll do it! Glad you found a solution :)
